packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -58,15 +57,62 @@ const SideNavCollapsedContainer = styled("div", {
     odysseyDesignTokens: DesignTokens;
     isSideNavCollapsed: boolean;
   }) => ({
-    backgroundColor: odysseyDesignTokens.HueNeutral300,
-    paddingTop: odysseyDesignTokens.Spacing5,
+    display: "flex",
+    "&:before": {
+      height: "100%",
+      width: 0,
+      content: '" "',
```

Comment:
Did we need the space or can we do with only the quotes?

```suggestion
      content: '""',
```

Typically, adding a space here has meaning. Not sure if that was intended or not.